### ISSUE TITLE
Expected color order of input image?

### ISSUE THREAD

**Issue by nyaang** [faustomorales/keras-facenet#1]
I am using the model named 20180408-102900-weights.h5(witch is trained on CASIA-WebFace dataset).The method **imread** in **OpenCV** is used to read image file, but the color order default is BGR. Could you please tell me which color order is expected to generate embeddings of images?BGR or RGB?

By the way, How do I choose the threshold for cosine similarity? I want to judge whether two different images belong to the same person.

**Comment by faustomorales** [comment]
Hi @nyaang -- The expected color order is RGB. I've added a note in the README just now to clarify that.

On choosing the threshold, it is definitely application-specific so it's best to have a test set on which to select a threshold based on your preference for precision / recall. I would build a distance matrix with your faces and see what distance starts to generate false positives. A good starting default is in the range of 0.55 - 0.6.

**Comment by nyaang** [comment]
> Hi @nyaang -- The expected color order is RGB. I've added a note in the README just now to clarify that.
> 
> On choosing the threshold, it is definitely application-specific so it's best to have a test set on which to select a threshold based on your preference for precision / recall. I would build a distance matrix with your faces and see what distance starts to generate false positives. A good starting default is in the range of 0.55 - 0.6.

Thank you very much
